xmppserver/src/main/java/org/jivesoftware/openfire/group/GroupNameInvalidException.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (C) 2004-2008 Jive Software. All rights reserved.
```

Comment:
When adding a new file, lets update the Copyright notice to this year and "Ignite Realtime Foundation"